File: C2203L/MVC/de11/Models/Employee.cs
using System.ComponentModel.DataAnnotations;

namespace de11.Models
{
    public class Employee
    {
        [Key]
        public int EmployeeNo { get; set; }
        public string EmployeeName { get; set; }
        public string Position { get; set; }
        public int Salary { get; set; }
    }
}

File: C2002LDoiCan/ADF1/04-03-2021/MyApp/src/myapp/MyApp/models/Employee.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package myapp.MyApp.models;

import java.util.Scanner;

public class Employee extends Person{
    private Float salary;
    private String position;
    Employee() {}
    public Employee(String name, String address) {
        super(name, address);
    }

    @Override
    public void input() {
        super.input(); 
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter salary: ");
        this.salary = scanner.nextFloat();
        System.out.println("Enter postition: ");
        this.position = scanner.next();
    }
    @Override
    public String toString() {
        return super.toString() + ",salary=" + salary 
                + ", position=" + position;
    }
}
